website/docs/installation.md

```diff
@@ -167,3 +167,8 @@ make install-with-models-metal
 ```
 
 Similarily, this ensures Metal devices are selected on model load, and Metal-specific kernels used when possible.
+
+### Supported OS Architectures
```

Comment:
We also support Windows x64.

Also these architecture name are the technical names - I would expect a more user-friendly name here (with the technical name in parenthesis, or if this was formatted as a table, in one of the columns)